.github/ISSUE_TEMPLATE/al-developer-experience-bug.md

```diff
@@ -13,6 +13,9 @@ Read the contributing https://github.com/microsoft/AL/blob/master/CONTRIBUTING.M
 Please follow this template in order for our developers to investigate your issue efficiently.
 
 Do not edit or remove the titles; e.g. "Descibe the bug".
```

Comment:
BTW there's a typo: "Desc**r**ibe the bug" :-)

Reply:
Thanks! :)